fem/quadinterpolator_grad_phys.hpp

```diff
@@ -0,0 +1,303 @@
+// Copyright (c) 2010-2020, Lawrence Livermore National Security, LLC. Produced
+// at the Lawrence Livermore National Laboratory. All Rights reserved. See files
+// LICENSE and NOTICE for details. LLNL-CODE-806117.
+//
+// This file is part of the MFEM library. For more information and source code
+// availability visit https://mfem.org.
+//
+// MFEM is free software; you can redistribute it and/or modify it under the
+// terms of the BSD-3 license. We welcome feedback and contributions, see file
+// CONTRIBUTING.md for details.
+
+#include "quadinterpolator.hpp"
+#include "../general/forall.hpp"
+#include "../linalg/dtensor.hpp"
+#include "../linalg/kernels.hpp"
+
+namespace mfem
+{
+
```

Comment:
These functions appear to be mostly (90%?) identical to the ones in quadinterpolator_grad.hpp. Would it be better to combine them with `if`-statements?

Reply:
We could use another templated argument to use one single header, that's possible.

Reply:
It could also be done for some other files.

Reply:
I combined the header files.